modules/ROOT/pages/quickstart.adoc

```diff
@@ -82,9 +82,9 @@ Packages follow a regular process from build to release, all managed by tags in
 [width="100%",cols="53%,47%",options="header",]
 |===
 |Koji Tag |Purpose
-|c9s-candidate |This is where new builds are stored, they will continue to c9s-gate
 |c9s-gate |This is where packages first land immediately after they build
-|c9s-pending |These packages have passed RHEL CI testing and CentOS Stream CI Testing (coming soon), these packages are also added to the buildroot when tagged here
+|c9s-candidate |These packages have passed gating based on RHEL CI testing and CentOS Stream CI Testing (coming soon)
+|c9s-pending |These packages have passed gating and preverification
```

Comment:
I like changing the description here, but we should keep the note about these packages being present in the buildroot. That's an important note for the external audience.

Reply:
I agree. I intended to keep it, but for some reason the note about buildroot was lost when moving sentences around. I've just fixed this.